I cannot run the tests:

```
$ ./mach test-wpt --release --ssl-type=none tests/wpt/mozilla/tests/mozilla/mozbrowser
Running 14 tests in web-platform-tests

Unsupported test type wdspec for product servo
  [0/14] /_mozilla/mozilla/mozbrowser/crossorigin_xhr.html
      1 [main] python2.7 5428 fork: child -1 - forked process 4696 died unexpectSTDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/crossorigin_xhr.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
  [0/14] /_mozilla/mozilla/mozbrowser/crossorigin_xhr.html
         /_mozilla/mozilla/mozbrowser/iframe_goback.html
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/crossorigin_xhr.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


STDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/iframe_goback.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
  [1/14] /_mozilla/mozilla/mozbrowser/iframe_reload_twice.html
         /_mozilla/mozilla/mozbrowser/iframe_goback.html
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/iframe_goback.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


STDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/iframe_reload_twice.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
  [2/14] /_mozilla/mozilla/mozbrowser/iframe_visibility.html
         /_mozilla/mozilla/mozbrowser/iframe_reload_twice.html
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/iframe_reload_twice.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


  [3/14] /_mozilla/mozilla/mozbrowser/iframe_visibility.html
      1 [main] python2.7 1244 fork: child -1 - forked process 3884 died unexpectSTDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/iframe_visibility.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
STDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/mozbrowser_click_fires_openwindow.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/iframe_visibility.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/mozbrowser_click_fires_openwindow.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


  [5/14] /_mozilla/mozilla/mozbrowser/mozbrowser_iframe_parentage.html
      1 [main] python2.7 3928 fork: child -1 - forked process 5232 died unexpectSTDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/mozbrowser_iframe_parentage.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/mozbrowser_iframe_parentage.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


  [6/14] /_mozilla/mozilla/mozbrowser/mozbrowser_loadevents.html
         /_mozilla/mozilla/mozbrowser/mozbrowsericonchange_event.html
      0 [main] python2.7 1172 fork: child -1 - forked process 5956 died unexpectedly, retry 0, exit code 0xC0000005, errno 11
STDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/mozbrowser_loadevents.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
STDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/mozbrowsericonchange_event.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/mozbrowsericonchange_event.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


  [7/14] /_mozilla/mozilla/mozbrowser/mozbrowser_loadevents.html
         /_mozilla/mozilla/mozbrowser/mozbrowserlocationchange_event.html
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/mozbrowser_loadevents.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


STDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/mozbrowserlocationchange_event.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
  [8/14] /_mozilla/mozilla/mozbrowser/mozbrowserlocationchange_event.html
         /_mozilla/mozilla/mozbrowser/mozbrowsersecuritychange_event.html
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/mozbrowserlocationchange_event.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


STDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/mozbrowsersecuritychange_event.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
  [9/14] /_mozilla/mozilla/mozbrowser/mozbrowsershowmodalprompt_event.html
         /_mozilla/mozilla/mozbrowser/mozbrowsersecuritychange_event.html
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/mozbrowsersecuritychange_event.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


STDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/mozbrowsershowmodalprompt_event.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/mozbrowsershowmodalprompt_event.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


  [11/14] /_mozilla/mozilla/mozbrowser/redirect.html
          /_mozilla/mozilla/mozbrowser/mozbrowsertitlechangedeagerly_event.html
STDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/mozbrowsertitlechangedeagerly_event.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
  [11/14] /_mozilla/mozilla/mozbrowser/redirect.html
          /_mozilla/mozilla/mozbrowser/mozbrowsertitlechangedeagerly_event.html
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/mozbrowsertitlechangedeagerly_event.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


STDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/redirect.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/redirect.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


  [13/14] /_mozilla/mozilla/mozbrowser/reload.html
      0 [main] python2.7 7792 fork: child -1 - forked process 2680 died unexpectSTDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/reload.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/reload.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


Ran 14 tests finished in 4.0 seconds.
  • 0 ran as expected. 0 tests skipped.
  • 14 tests had errors unexpectedly
```

--- a/components/script/dom/htmliframeelement.rs
+++ b/components/script/dom/htmliframeelement.rs
@@ -379,9 +379,9 @@ impl MozBrowserEventDetailBuilder for HTMLIFrameElement {
             MozBrowserEvent::TitleChange(ref string) => {
                 string.to_jsval(cx, rval);
             }
-            MozBrowserEvent::LocationChange(uri, can_go_back, can_go_forward) => {
+            MozBrowserEvent::LocationChange(url, can_go_back, can_go_forward) => {
                 BrowserElementLocationChangeEventDetail {
-                    uri: Some(DOMString::from(uri)),
+                    url: Some(DOMString::from(url)),
                     canGoBack: Some(can_go_back),
                     canGoForward: Some(can_go_forward),
                 }.to_jsval(cx, rval);

--- a/components/script/dom/webidls/BrowserElement.webidl
+++ b/components/script/dom/webidls/BrowserElement.webidl
@@ -64,7 +64,7 @@ dictionary BrowserElementErrorEventDetail {
 };
 
 dictionary BrowserElementLocationChangeEventDetail {
-  DOMString uri;
+  DOMString url;
   boolean canGoBack;
   boolean canGoForward;
 };

--- a/tests/wpt/mozilla/tests/mozilla/mozbrowser/iframe_goback.html
+++ b/tests/wpt/mozilla/tests/mozilla/mozbrowser/iframe_goback.html
@@ -19,8 +19,8 @@ async_test(function(t) {
   iframe.src = url1;
 
   iframe.addEventListener("mozbrowserlocationchange", e => {
-    locations.push(e.detail.uri);
-    if (e.detail.uri == url2) {
+    locations.push(e.detail.url);
+    if (e.detail.url == url2) {
       iframe.goBack();
     }
     if (locations.length == expected_locations.length) {

--- a/tests/wpt/mozilla/tests/mozilla/mozbrowser/mozbrowserlocationchange_event.html
+++ b/tests/wpt/mozilla/tests/mozilla/mozbrowser/mozbrowserlocationchange_event.html
@@ -39,7 +39,7 @@ async_test(function(t) {
   var action_idx = 0;
 
   iframe.addEventListener("mozbrowserlocationchange", e => {
-    received_events.push(e.detail.uri);
+    received_events.push(e.detail.url);
     received_events.push(e.detail.canGoBack);
     received_events.push(e.detail.canGoForward);
 

--- a/tests/wpt/mozilla/tests/mozilla/mozbrowser/redirect.html
+++ b/tests/wpt/mozilla/tests/mozilla/mozbrowser/redirect.html
@@ -10,7 +10,7 @@
         iframe.mozbrowser = "true";
         iframe.src = "redirect_init.html?pipe=status(302)|header(Location,redirect_final.html)";
         iframe.addEventListener("mozbrowserlocationchange", t.step_func(e => {
-            assert_equals(e.detail.uri, new URL("redirect_final.html", location).href);
+            assert_equals(e.detail.url, new URL("redirect_final.html", location).href);
             t.done();
         }));
         document.body.appendChild(iframe);

